.github/README.md

```diff
@@ -14,7 +14,7 @@ $ go build -o terraform-provider-datadome
 
 1. Build the provider.
 
-2. Install it.
+2. Install it (you may need to change the value of `OS_ARCH` inside Makefile).
```

Comment:
Can you list the main values to use? (I guess it can be something like `linux_amd64`, `darwin_amd64` and `win_amd64`?)

Reply:
@darahak-datadome : I have changed the README to add a step. The main values you listed are set as comments inside the Makefile so that we only have to uncomment the value we want to use.